app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -623,6 +623,7 @@ def log_enrollment_updated_analytics(enrollment:, enrollment_passed:, enrollment
       reason: reason,
       job_name: self.class.name,
       tmx_status: enrollment.profile&.tmx_status,
+      profile_age_in_seconds: enrollment.profile&.profile_age_in_seconds,
```

Comment:
Should the `profile_age` be logged anytime the enrollment changes from `pending` status to a completed status? If so the `get_usps_proofing_results_job` needs to updated to log the `profile_age` when it changes the enrollment to `cancelled`, `expired`, and `failed` statuses.

Reply:
Actually, I am wrong here... by updating this method it should capture all of these status updates. I was thinking there was another way were logging a status change else where in the file. This looks good to me.